osu.Game/Overlays/OptionsOverlay.cs

```diff
@@ -33,10 +33,13 @@ public class OptionsOverlay : OverlayContainer
 
         private ScrollContainer scrollContainer;
         private OptionsSidebar sidebar;
+        private OptionsSidebar.SidebarButton[] sidebarButtons;

```

Comment:
i'd rather you just move the class out to a higher level than use this kind of class reference.